typescript/sdk/src/app.ts

```diff
@@ -38,7 +38,7 @@ export class AbacusApp<
     return this.get(chain).addresses;
   }
 
-  reconnect(chain: Chain, connection: Connection) {
+  reconnect(chain: Chain, connection: Connection): void {
```

Comment:
Do method signatures now always have to be explicit?

Reply:
Only in the SDK package